tests/test-utils.mjs

```diff
@@ -53,11 +53,11 @@ class SequenceVerifier {
     this.index = 0;
   }
   check(toCheck, elementCount = toCheck.length, offset = 0) {
-    if (typeof toCheck === Number) {
+    if (typeof toCheck === "number") {
```

Comment:
Was this intended originally? Would changing this lead to diverging test behaviour?